File: app/src/main/java/com/example/TeethHealth/Diseases.java
package com.example.TeethHealth;

import android.graphics.Bitmap;
import android.widget.TextView;

public class Diseases{

    private final Bitmap bitmap;
    private final TextView tvStatus;

    public Diseases(Bitmap _bitmap, TextView _tvStatus)
    {
        this.bitmap = _bitmap;
        this.tvStatus = _tvStatus;
    }
    public static TextView GetTvStatus(Diseases dise)
    {
        return dise.tvStatus;
    }
    public static Bitmap GetBitmap(Diseases dise)
    {
        return dise.bitmap;
    }
}

File: app/src/main/java/com/example/TeethHealth/ImageFilters/Diseases.kt
package com.example.TeethHealth.ImageFilters

import android.graphics.Bitmap
import android.widget.TextView

class Diseases(private val bitmap: Bitmap?, private val tvStatus: TextView?) {
    companion object {
        fun GetTvStatus(dise: Diseases?): TextView? {
            return dise?.tvStatus
        }

        fun GetBitmap(dise: Diseases?): Bitmap? {
            return dise?.bitmap
        }
    }
}